index.html

```diff
@@ -1,111 +1,424 @@
+                    One September 22, 2021 A volutpat nulla eros, nulla ultrices
+                    nec nulla imperdict
+                  </p>
+                </article>
+                <button>Read more ></button>
               </div>
-
-              <div class="name">
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <p>error!!!!</p>
-                <button class="ok-error-btn" onclick="window.location.reload();">OK...</button>
+            </div>
+          </div>
+          <div class="post">
+            <div>
+              <img src="#" alt="text" />
+            </div>
+            <div>
+              <div>
+                <p><span class="folder icon"></span>Tony Nguyen</p>
+                <p><span class="folder icon"></span>20 Jun 2022</p>
               </div>
+              <article>
+                <h1>Sonic live</h1>
+                <p>
+                  One September 22, 2021 A volutpat nulla eros, nulla ultrices
+                  nec nulla imperdict
+                </p>
+              </article>
+              <button>Read more ></button>
             </div>
           </div>
         </div>
-      </template>
-      <!-- <script src="script.js"></script> --> <!-- uncomment script --> 
-</body>
+        <button>Read more</button>
+      </section>
+      <section class="about-as">
+        <div>
+          <div>
+            <h1>About us</h1>
+            <p>
+              Duls aute irure dolor in reprehenderit in voluptate velit esse
+              cillum dolore eu fugiat nulla pariatur. Excepteur sint occae cat
+              cupidatat not prodient, sunt in culpa qui officia dese runt molit
+              anim id est laborum velit esse cillium dolore eu fugiat nulla
+              pariatu epteur sint occaecat.
+            </p>
+          </div>
+          <div>
+            <h1>High quality</h1>
+            <p>
+              Lorem ipsum is simply dummy text of the printing and ipsum
+              typesetting industry.
+            </p>
+          </div>
+          <div>
+            <h1>Big collection</h1>
+            <p>
+              Lorem ipsum is simply dummy text of the printing and ipsum
+              typesetting industry.
+            </p>
+          </div>
+          <div>
+            <h1>Top resourse</h1>
+            <p>
+              Lorem ipsum is simply dummy text of the printing and ipsum
+              typesetting industry.
+            </p>
+          </div>
+          <div>
+            <h1>Big Community</h1>
+            <p>
+              Lorem ipsum is simply dummy text of the printing and ipsum
+              typesetting industry.
+            </p>
+          </div>
+        </div>
+        <button>Read more</button>
+      </section>
+      <section class="our-team">
+        <div>
+          <h1>Our team</h1>
+        </div>
+        <div>
+          <div class="avatar">
```

Comment:
This name for the block is not very clear. If you look at the design, this block is one of the seven parts of the parent element. This should be reflected in the class name for example, 'our-team-item' or 'team-item'

Reply:
done